src/model.jl

```diff
@@ -242,6 +242,13 @@ function create_model(graph, params, sets; verbose = false, write_lp_file = fals
         params.initial_storage_capacity[a] + (a ∈ Ai ? energy_limit[a] : 0.0)
     )
 
+    # - cycling condition for storage level
+    @constraint(
+        model,
+        cycling_condition_for_storage_level[a ∈ As, rp ∈ RP, T ∈ P[(a, rp)]],
+        storage_level[a, rp, T] ≥ params.initial_storage_level[a]
```

Comment:
this should be done with set_lower_bound()

Reply:
I agree. The new commit has the changes